pkgs/development/python-modules/magic-wormhole/default.nix

```diff
@@ -22,30 +21,25 @@
 
 buildPythonPackage rec {
   pname = "magic-wormhole";
-  version = "0.10.4";
-  name = "${pname}-${version}";
+  version = "0.10.5";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "cd3105975e71bc6437848c7fc9f0b23ef0e0c625c8b19ec66a5ddc727c6d11ae";
+    sha256 = "9558ea1f3551e535deec3462cd5c8391cb32ebb12ecd8b40b36861dbee4917ee";
   };
 
   checkInputs = [ mock ];
-  buildInputs = [ nettools glibcLocales ];
+  buildInputs = [ nettools ];
   propagatedBuildInputs = [ autobahn cffi click hkdf pynacl spake2 tqdm ipaddress humanize pyopenssl service-identity txtorcon ];
 
   postPatch = ''
     sed -i -e "s|'ifconfig'|'${nettools}/bin/ifconfig'|" src/wormhole/ipaddrs.py
     sed -i -e "s|if (os.path.dirname(os.path.abspath(wormhole))|if not os.path.abspath(wormhole).startswith('/nix/store') and (os.path.dirname(os.path.abspath(wormhole))|" src/wormhole/test/test_cli.py
-  '' + lib.optionalString (pythonAtLeast "3.3") ''
```

Comment:
Also explaining why this was removed would be nice.

Reply:
Because of https://github.com/warner/magic-wormhole/commit/13b1539409b0afc1e63ad292322d64bd18e35eb0, which happened after the 0.10.5 release as I see right now.
I can however revert that change if you want.